### PR TITLE
Add dependabot.yml to detect new rf releases #436

### LINKED ISSUE #436: Automatically detect new Robot Framework version
If there is new version of Robot Framework released, open new issue in Robocop with title "Check compatibility with Robot Framework X" (X = version) or similar. Link to release notes in content of issue. 

This will allow us to detect new version and prepare Robocop for them. 

It's up to discussion if we should detect official versions (such as 4.1.1) or release candidates (4.1.1rc1) so we have extra time before release go live.

### PR DESCRIPTION
**Pull request by MalikMlitat**
Closes #436

### CODE REVIEW

File: tests/packages/recent/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+robotframework==3v4.1.2rc2
```
Comment (bhirsz) [inline tests/packages/recent/requirements.txt:1]: What version is it?

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install robotframework${{ matrix.rf-version }}
+        pip install -r /tests/packages/{{ matrix.rf-version }}/requirements.txt
```
Comment (bhirsz) [inline .github/workflows/unit-tests.yml:46]: Error from checks: `Hint: It looks like a path. File '}}/requirements.txt' does not exist.`. Most likely missing `$` before `{{`

File: tests/packages/recent/requirements.txt (hunk 1 above)
Comment (MalikMlitat) [reply tests/packages/recent/requirements.txt:1]: this was only for test purposes, just to check if dependabot would start a PR to the latest version
https://github.com/MalikMlitat/robotframework-robocop/pulls

File: .github/workflows/unit-tests.yml (hunk 2 above)
Comment (MalikMlitat) [reply .github/workflows/unit-tests.yml]: done

File: tests/packages/recent/requirements.txt (hunk 1 above)
Comment (MalikMlitat) [reply tests/packages/recent/requirements.txt]: replaces now with latest (4.1.2)

**Review by bhirsz — APPROVED** [review]
Looks good! I will also reuse the requirements files later because we specify requirements in few places (docs, tox config etc) and it's all over the place.

**Comment by MalikMlitat** [comment]
@bhirsz you have to enable/activate dependabot for the project

**Comment by bhirsz** [comment]
> @bhirsz you have to enable/activate dependabot for the project

Looks like it's working:
image

**Pull request merged**